templates/ufw/user6.rules.j2

```diff
@@ -0,0 +1,69 @@
+# {{ ansible_managed }}
+
+*filter
+:ufw6-user-input - [0:0]
+:ufw6-user-output - [0:0]
+:ufw6-user-forward - [0:0]
+:ufw6-before-logging-input - [0:0]
+:ufw6-before-logging-output - [0:0]
+:ufw6-before-logging-forward - [0:0]
+:ufw6-user-logging-input - [0:0]
+:ufw6-user-logging-output - [0:0]
+:ufw6-user-logging-forward - [0:0]
+:ufw6-after-logging-input - [0:0]
+:ufw6-after-logging-output - [0:0]
+:ufw6-after-logging-forward - [0:0]
+:ufw6-logging-deny - [0:0]
+:ufw6-logging-allow - [0:0]
+:ufw6-user-limit - [0:0]
+:ufw6-user-limit-accept - [0:0]
+### RULES ###
+
+### tuple ### limit tcp {{ bbb_ssh_port }} ::/0 any ::/0 in
+-A ufw6-user-input -p tcp --dport {{ bbb_ssh_port }} -m conntrack --ctstate NEW -m recent --set
+-A ufw6-user-input -p tcp --dport {{ bbb_ssh_port }} -m conntrack --ctstate NEW -m recent --update --seconds 30 --hitcount 6 -j ufw6-user-limit
+-A ufw6-user-input -p tcp --dport {{ bbb_ssh_port }} -j ufw6-user-limit-accept
+
+### tuple ### allow tcp 443 ::/0 any ::/0 in
+-A ufw6-user-input -p tcp --dport 443 -j ACCEPT
+
+### tuple ### allow tcp 80 ::/0 any ::/0 in
+-A ufw6-user-input -p tcp --dport 80 -j ACCEPT
+
+{% if (bbb_coturn_enable | bool) %}
+### tuple ### allow any {{ bbb_coturn_port }} ::/0 any ::/0 in
+-A ufw6-user-input -p tcp --dport {{ bbb_coturn_port }} -j ACCEPT
+-A ufw6-user-input -p udp --dport {{ bbb_coturn_port }} -j ACCEPT
+
+### tuple ### allow any {{ bbb_coturn_port_tls }} ::/0 any ::/0 in
+-A ufw6-user-input -p tcp --dport {{ bbb_coturn_port_tls }} -j ACCEPT
+-A ufw6-user-input -p udp --dport {{ bbb_coturn_port_tls }} -j ACCEPT
+
+### tuple ### allow udp {{ bbb_coturn_min_port }}:{{ bbb_coturn_max_port }} ::/0 any ::/0 in
+-A ufw6-user-input -p udp -m multiport --dports {{ bbb_coturn_min_port }}:{{ bbb_coturn_max_port }} -j ACCEPT
+
+{% endif %}
+### tuple ### allow udp {{ bbb_freeswitch_min_port }}:{{ bbb_freeswitch_max_port }} ::/0 any ::/0 in
```

Comment:
Hi @ChrisKar96, I think you forgot to remove the variable `bbb_freeswitch_min_port` also from the comments :)

Reply:
Nice catch again! Fixed it. I figured that we didn't really need to have new variables for the freeswitch ports, so I opted to revert that change.